Composer/packages/extensions/obiformeditor/src/Form/widgets/LgEditorWidget.tsx

```diff
@@ -70,6 +66,18 @@ export const LgEditorWidget: React.FC<LgEditorWidgetProps> = props => {
     body: getInitialTemplate(name, value),
   };
 
+  const lgStartLine = (lgFile && lgFile.content.split('\n').findIndex(item => item.trim() === `# ${lgName}()`)) || -1;
```

Comment:
we should get template start line from lgParser, it would be more reliable

Reply:
yes, let's mark an issue with this.